web-server/src/components/Service/SystemStatus.tsx

```diff
@@ -99,72 +91,67 @@ export const SystemStatus: FC = () => {
-                  position: 'relative',
-                  overflow: 'hidden'
+                    isUp ? borderColor : theme.colors.error.main
+                  }`
                 }}
               >
                 <FlexBox col flexGrow={1} minHeight="5em">
                   <FlexBox alignCenter justifyContent="space-between">
                     <Line
                       white
                       bold
-                      sx={{
-                        fontSize: '1.2em',
-                        display: 'flex',
-                        alignItems: 'center'
-                      }}
+                      fontSize={'1.2em'}
+                      display={'flex'}
+                      alignItems={'center'}
                     >
                       {serviceTitle[serviceKey]}
-                      <Box
+                      <FlexBox
                         component="span"
-                        sx={{
-                          display: 'inline-block',
-                          marginLeft: '6px',
-                          width: '10px',
-                          height: '10px',
-                          borderRadius: '50%',
-                          backgroundColor: isUp
+                        marginLeft={'6px'}
+                        borderRadius={'50%'}
+                        width={'10px'}
+                        height={'10px'}
+                        bgcolor={
+                          isUp
                             ? theme.colors.success.main
                             : theme.colors.error.main
-                        }}
+                        }
                       />
                     </Line>
                   </FlexBox>
 
                   <FlexBox col relative fullWidth flexGrow={1}>
-                    <FlexBox
-                      alignCenter
-                      sx={{ width: '100%', paddingTop: '8px' }}
-                    >
+                    <FlexBox alignCenter fullWidth paddingTop={'8px'}>
                       <Line
-                        sx={{
-                          fontWeight: '500',
-                          fontSize: '0.95em',
-                          color: isUp ? '#28a745' : '#dc3545',
-                          lineHeight: '1.4'
-                        }}
+                        fontWeight={'500'}
+                        fontSize={'0.95em'}
+                        lineHeight={'1.4'}
+                        white
+                      >
+                        Status:
+                      </Line>
+                      <Line
+                        fontWeight={'500'}
+                        fontSize={'0.95em'}
+                        lineHeight={'1.4'}
+                        color={
+                          isUp
+                            ? theme.colors.success.main
+                            : theme.colors.error.main
+                        }
+                        marginLeft={'8px'}
                       >
```

Comment:
@VipinDevelops prefer to use something like `<Line bold bigish white>`

Reply:
Ohh yeah fogot this one